README.md

```diff
@@ -6,30 +6,28 @@ An _experimental_ (Read: not-usable or in anyway done) distributed podcasting pl
 
 Getting started:
 
-Install the vendor dependencies using godep. (https://github.com/golang/dep)
```

Comment:
FIX: Keep this part in, it helps document to less-experienced go folks (like myself 2 days ago 😅) that we're using godep and not govendor.